src/Common/src/Common/Expression/CSharp/ExpressionParser.cs

```diff
@@ -1,21 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the Apache 2.0 License.
-// See the LICENSE file in the project root for more information.
-
-using System;
-
-namespace Steeltoe.Common.Expression.CSharp
-{
-    public class ExpressionParser : IExpressionParser
```

Comment:
Is a CSharp expression parser something we are still planning to add in the future? I don't see it in the backlog

Reply:
Not at this point